data/scripts/lib/create_functions.lua

```diff
@@ -1,2 +1,4 @@
 createFunctions(MonsterType) -- creates get/set functions for MonsterType
 createFunctions(Spell) -- creates get/set functions for Spell
+
+dofile('data/scripts/spells/lib/spells.lua')
```

Comment:
this doesn't need to be here, just move the file `spells.lua` into `scripts/lib` it will be automaticly load before anything else.

Reply:
You forgot to remove this `dofile` part in your latest commit